qmlui/qlcplus_de_DE.ts

```diff
@@ -0,0 +1,4150 @@
+        <source>Full</source>
+        <translation>Alle</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorTool.qml" line="96"/>
+        <source>Filters</source>
+        <translation>Farbfilter</translation>
+    </message>
+</context>
+<context>
+    <name>ColorToolBasic</name>
+    <message>
+        <location filename="qml/ColorToolBasic.qml" line="192"/>
+        <source>Selected color</source>
+        <translation>Ausgewählte Farbe</translation>
+    </message>
+</context>
+<context>
+    <name>ColorToolFilters</name>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="127"/>
+        <source>Open filters menu</source>
+        <translation>Filter-Menü öffnen</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="209"/>
+        <source>Cyan</source>
+        <translation>Cyan</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="209"/>
+        <source>Red</source>
+        <translation>Rot</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="229"/>
+        <source>White</source>
+        <translation>Weiß</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="250"/>
+        <source>Magenta</source>
+        <translation>Magenta</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="250"/>
+        <source>Green</source>
+        <translation>Grün</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="270"/>
+        <source>Amber</source>
+        <translatorcomment>Nicht &quot;Bernstein&quot;, da &quot;Amber&quot; auch im Deutschen die übliche LED-Bezeichnung ist.</translatorcomment>
+        <translation>Amber</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="291"/>
+        <source>Yellow</source>
+        <translation>Gelb</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="291"/>
+        <source>Blue</source>
+        <translation>Blau</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="311"/>
+        <source>UV</source>
+        <translation>UV</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="332"/>
+        <source>CMY</source>
+        <translation>CMY</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="379"/>
+        <source>Add a new color filters file</source>
+        <translation>Neue Farbfilterdatei hinzufügen</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="393"/>
+        <source>Rename the current color filters</source>
```

Comment:
@mcallegari Is this actually "Rename the current color filters _file_"? Or does it refer to a single filter and not the filter file?

Reply:
Yeah, it refers to the file, not a single filter. I need to review that part cause it didn't look right to me either. For the time being, if unsure, just leave the string untranslated.